types/diamorphosis.d.ts

```diff
@@ -0,0 +1,12 @@
+declare interface DiamorphosisInput {
+  configFolder?: string;
+  configPath?: string;
+  envFolder?: string;
+  loadDotEnv?: string;
+}
+
+declare module 'diamorphosis' {
```

Comment:
I am not sure this will work. 
Check here: https://github.com/SokratisVidros/race-until/blob/master/src/race-until.d.ts